src/Dot/props.tsx

```diff
@@ -0,0 +1,8 @@
+import CSS from 'csstype';
+
+export interface Props {
+  avatarUrl?: string,
+  username?: string,
+  style?: CSS.Properties,
+  visible: boolean
```

Comment:
Yeah I'd make this not required.

When people setup a dot it should just show unless o.t.w specified